### DISCUSSION TITLE
brew install pianobar failed

### DISCUSSION THREAD

**Discussion by governor22** [Homebrew/discussions#4779] [Getting started]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: 12c8778af976269344f3c0b914243e178fc4ffbd
Last commit: 3 days ago
Core tap JSON: 10 Sep 04:26 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: octa-core 64-bit haswell
Clang: 10.0.1 build 1001
Git: 2.20.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.54.0 => /usr/bin/curl
macOS: 10.14.6-x86_64
CLT: 10.3.0.0.1.1562985497
Xcode: 11.3.1
```

### Output of `brew doctor`

```shell
Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences.

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 11.3.1.


Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libFLAC++.2.1.2.dylib
  /usr/local/lib/libFLAC++.2.dylib
  /usr/local/lib/libFLAC++.dylib
  /usr/local/lib/libFLAC.4.1.2.dylib
  /usr/local/lib/libFLAC.4.dylib
  /usr/local/lib/libFLAC.dylib
  /usr/local/lib/libOggFLAC++.0.0.4.dylib
  /usr/local/lib/libOggFLAC++.0.dylib
  /usr/local/lib/libOggFLAC++.dylib
  /usr/local/lib/libOggFLAC.1.0.2.dylib
  /usr/local/lib/libOggFLAC.1.dylib
  /usr/local/lib/libOggFLAC.dylib
  /usr/local/lib/libmp3lame.0.0.0.dylib
  /usr/local/lib/libmp3lame.0.dylib
  /usr/local/lib/libmp3lame.dylib
  /usr/local/lib/libogg.0.5.0.dylib
  /usr/local/lib/libogg.0.dylib
  /usr/local/lib/libogg.dylib
  /usr/local/lib/libvorbis.0.3.0.dylib
  /usr/local/lib/libvorbis.0.dylib
  /usr/local/lib/libvorbis.dylib
  /usr/local/lib/libvorbisenc.2.0.0.dylib
  /usr/local/lib/libvorbisenc.2.dylib
  /usr/local/lib/libvorbisenc.dylib
  /usr/local/lib/libvorbisfile.3.1.0.dylib
  /usr/local/lib/libvorbisfile.3.dylib
  /usr/local/lib/libvorbisfile.dylib

Warning: Unbrewed '.la' files were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected '.la' files:
  /usr/local/lib/libFLAC++.la
  /usr/local/lib/libFLAC.la
  /usr/local/lib/libOggFLAC++.la
  /usr/local/lib/libOggFLAC.la
  /usr/local/lib/libmp3lame.la
  /usr/local/lib/libogg.la
  /usr/local/lib/libvorbis.la
  /usr/local/lib/libvorbisenc.la
  /usr/local/lib/libvorbisfile.la

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libFLAC++.a
  /usr/local/lib/libFLAC.a
  /usr/local/lib/libOggFLAC++.a
  /usr/local/lib/libOggFLAC.a
  /usr/local/lib/libmp3lame.a
  /usr/local/lib/libogg.a
  /usr/local/lib/libvorbis.a
  /usr/local/lib/libvorbisenc.a
  /usr/local/lib/libvorbisfile.a
```

### Description of issue

I am trying to install pianobar (a Pandora music client for terminal) using homebrew. The error I got was:

_Error: pianobar: Failed to download resource "libvorbis"
Download failed: Homebrew-installed 'curl' is not installed for: https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz_

Please bear with me, I'm not a dev and I don't code. I'm running a 2015 MBP with MacOS Mojave. 
Can someone steer me on what I need to do?
Thank you

**Comment by ZhongRuoyu** [comment]
This means that the resources needs to be downloaded with a Homebrew-installed `curl`. Please run `brew install curl`, and try again.

**Reply by governor22** [reply]
@ZhongRuoyu  Thank you for your quick reply.  I ran "brew install curl".   Then I ran "brew install pianobar" again. 

There is a new error this time:


==> cmake --build builddir
Last 15 lines from /Users/Josh2/Library/Logs/Homebrew/highway/02.cmake:
/tmp/highway-20230910-36601-ezm3g0/highway-1.0.7/hwy/ops/x86_256-inl.h:5781:33: error: use of undeclared identifier '_knot_mask8'
      _mm256_maskz_cvttpd_epu32(_knot_mask8(MaskFromVec(v).raw), v.raw)};
                                ^
In file included from /tmp/highway-20230910-36601-ezm3g0/highway-1.0.7/hwy/per_target.cc:20:
In file included from /tmp/highway-20230910-36601-ezm3g0/highway-1.0.7/hwy/foreach_target.h:127:
In file included from /tmp/highway-20230910-36601-ezm3g0/highway-1.0.7/hwy/per_target.cc:21:
In file included from /tmp/highway-20230910-36601-ezm3g0/highway-1.0.7/hwy/highway.h:410:
/tmp/highway-20230910-36601-ezm3g0/highway-1.0.7/hwy/ops/x86_512-inl.h:4947:33: error: use of undeclared identifier '_knot_mask8'
      _mm512_maskz_cvttpd_epu32(_knot_mask8(MaskFromVec(v).raw), v.raw)};
                                ^
**4 errors generated.
make[2]: *** [CMakeFiles/hwy.dir/hwy/per_target.cc.o] Error 1
make[1]: *** [CMakeFiles/hwy.dir/all] Error 2
make: *** [all] Error 2**


Do you know what this means?

**Reply by governor22** [reply]
OK - I ended up installing pianobar with Macports.  Problem solved.  And pianobar working!
